core/src/main/java/org/geysermc/geyser/translator/inventory/item/ItemTranslator.java

```diff
@@ -118,7 +123,7 @@ private static ItemData.Builder translateToBedrock(GeyserSession session, Item j
         nbt = translateDisplayProperties(session, nbt, bedrockItem);
 
         if (nbt != null) {
```

Comment:
While we're at it - we should check the number tag HideFlags and see if the bitmask contains 2 - if it does, this code should not run.